src/main/java/mate/academy/springboot/datajpa/service/ProductService.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.springboot.datajpa.service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.datajpa.model.Product;
+
+public interface ProductService {
+    Product save(Product product);
+
+    Product getById(Long id);
+
+    void delete(Long id);
+
+    Product update(Product product);
+
+    List<Product> getAllWherePriceBetween(BigDecimal priceMin, BigDecimal priceMax);
+
+    List<Product> getAllInCategories(Long categoryId);
```

Comment:
Naming with "in" is usually used when we have List as argument to indicate we check all values in list. So such naming here is slightly confusing
```suggestion
    List<Product> getAllByCategory(Long categoryId);
```

Reply:
fixed